src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -463,14 +453,14 @@ private string EscapeQuery(string format, params string[] args)
 
         private void Dispose(bool disposing)
         {
-            if (!this.disposedValue)
+            if (!disposedValue)
```

Comment:
The class is sealed, we can remove almost all of this boilerplate (not in this PR)